libs/feature/record/src/lib/state/mdview.facade.ts

```diff
@@ -52,6 +52,11 @@ export class MdViewFacade {
     filter((md) => !!md)
   )
 
+  featureCatalog$ = this.store.pipe(
+    select(MdViewSelectors.getFeatureCatalog),
+    filter((fc) => !!fc)
```

Comment:
I would remove the `filter` here as it can be misleading for a consumer of this stream; e.g. if no feature catalog was found, this stream would never emit a value